docs/getting-started/01-overview.md

```diff
@@ -0,0 +1,81 @@
+
+## Steps
+
+These guides cover the following steps:
+
+1. [Create a Namespace](#getting-started-create-a-namespace) (`orders-service`) in which you will deploy all resources.
+2. [Create a microservice](#getting-started-deploy-a-microservice) by deploying `orders-service` on the cluster in the `orders-service` Namespace.
+3. [Expose the microservice](#getting-started-expose-the-microservice) through the APIRule CR on HTTP endpoints. This way it will be available for other services outside the cluster so you could retrieve data from it and send sample orders to it. Since the microservice uses in-memory storage, when you send a sample order to its endpoint, the order details will be gone after you restart the microservice.
+4. [Add the Redis service](#getting-started-add-the-redis-service) as an addon.
+5. [Create the Redis service instance](#getting-started-create-a-service-instance-for-the-redis-service) in the Namespace (ServiceInstance CR) so you can bind it with the microservice and Function.
+6. [Bind the microservice to the Redis service](#getting-started-bind-the-redis-service-instance-to-the-microservice) by creating ServiceBinding and ServiceBindingUsage CRs. Send a sample order to its endpoint - since it now uses the Redis storage, the order details will not be gone after you restart the microservice.
+7. [Connect Commerce mock](#getting-started-connect-an-external-application) as the external application.
+8. [Trigger the microservice](#getting-started-trigger-the-microservice-with-an-event) to react to the `order.deliverysent.v1` event from the Commerce mock. Send the event and see if the microservice reacts to it by saving its details in the Redis database.
+9. [Create a Function](#getting-started-create-a-function) in the Namespace and repeat the microservice flow:
+- [Expose the Function](#getting-started-expose-the-function) through the APIRule CR on HTTP endpoints. This way it will be available for other services outside the cluster.
+- [Bind a Function to the Redis service](#getting-started-bind-a-redis-service-instance-to-the-function) by creating ServiceBinding and ServiceBindingUsage CRs.
+- [Trigger the Function](#getting-started-trigger-the-function-with-an-event) to react to the `order.deliverysent.v1` event from Commerce mock. Send the event and see if the Function reacts to it by saving its details in the Redis database.
+
+As a result, you get two scenarios of the same flow - a microservice and a Function that are triggered by new order events from Commerce mock and send order data to the Redis database:
```

Comment:
```suggestion
As a result, you get two scenarios of the same flow — a microservice and a Function that are triggered by new order events from the Commerce mock and send the order data to the Redis database:
```
I switched the hyphen (`-`) to the m-dash (`—`), but this is not clearly visible in the monospace-font suggestion.

Reply:
I changed everything except for `the` in front of Commerce mock.

Reply:
Sure :)